frontend/sass/_text.scss

```diff
@@ -2,7 +2,32 @@ body {
   color: $text-color;
   font-family: $sans-serif;
 }
+// Scott start styles
```

Comment:
do we need to keep these wrapper comments around these styles?

Reply:
No - that was for my WIP, thanks for finding that. I'd like to move the styles where they belong.

Reply:
@line47 - can you make a quick PR to remove these comments and move the styles to where they belong?